### PR TITLE
chore: add wip errata doc for v2

### PR DESCRIPTION
**Pull request by bharathkkb**

### CODE REVIEW

File: ERRATA.md [hunk 1]
```diff
@@ -1,19 +1,40 @@
...
+- Splunk log integration will be integrated in a future release.
+- Cloud Asset Inventory will be integrated in a future release.
+- The unallocated IP address space in the Shared VPC networks, described in Section 7.3, is currently being used by Private Service Networking in this release.
+
+## [1.x](https://github.com/terraform-google-modules/terraform-example-foundation/releases/tag/v1.0.0)
+### Code Discrepancies
+
+#### Labeling
+- The guide defines vpc-type for shared, service, float, nic, and peer projects. It does not define a vpc-type for Jenkins agents (vpc-b-jenkinsagents), the DNS Hub (vpc-dns-hub) and projects created in 4-projects.
+This will be addressed in the next version of the whitepaper.
```
Comment (daniel-cit) [inline ERRATA.md:29]:
```suggestion
This will be addressed in the next version of the blueprint guide.
```

File: ERRATA.md [hunk 2]
```diff
@@ -1,19 +1,40 @@
...
+# Errata Summary
 This is an overview of the delta between the example foundation repository and the [Google Cloud security foundations guide](https://services.google.com/fh/files/misc/google-cloud-security-foundations-guide.pdf), including code discrepancies and notes on future automation. This document will be updated as new code is merged.
 
+## 2.x [WIP]
 ### Code Discrepancies
 
-##### Labeling
+#### Labeling
 - The guide defines vpc-type for shared, service, float, nic, and peer projects. It does not define a vpc-type for Jenkins agents (vpc-b-jenkinsagents), the DNS Hub (vpc-dns-hub) and projects created in 4-projects.
 This will be addressed in the next version of the whitepaper.
```
Comment (daniel-cit) [inline ERRATA.md:9]:
```suggestion
This will be addressed in the next version of the blueprint guide.
```

**Review by daniel-cit — COMMENTED** [review]
LGTM.
I would unify all references to the document as `guide`.

**Review by rjerrems — APPROVED** [review]
LGTM

**Pull request merged**